### DISCUSSION TITLE
Different color schemes for different tabs

### DISCUSSION THREAD

**Discussion by seh** [kovidgoyal/kitty#6512] [Q&A] [answered]
I've searched previous discussions, but didn't find any that addressed this desire: _I'd like to use different color schemes (or themes) in different tabs within the same window._ I've been using this approach in macOS's _Terminal.app_ for many years, as it helps me distinguish quickly different tabs focusing on different tasks.

So far in my experiments with _kitty_ it appears that the theme-related commands such as `background` and `foreground` operate at either the process-wide or window level, and aren't scoped to particular tabs. The `set-colors` command appears to change the colors in all the tabs of all the windows of the _kitty_ process responding to remote control.

Is there any means by which I can specific a color scheme to use for one tab, and then use a different scheme for a different tab?

**Comment by kovidgoyal** [comment, marked as answer]
No, color schemes can be set per kitty window (not OS window) or
globally. See kitty @ set-colors -h

**Reply by seh** [reply]
Thank you. I had not understood the containment relationship between OS windows, tabs, and _kitty_ windows, but now I see that it's possible to emulate my goal closely enough using windows rather than tabs.

I'll keep experimenting with this. In the meantime, thank you for this amazing program.
